contracts/votingMachines/GenesisProtocolLogic.sol

```diff
@@ -216,6 +217,10 @@ contract GenesisProtocolLogic is IntVoteInterface {
 
     /**
       * @dev executeBoosted try to execute a boosted or QuietEndingPeriod proposal if it is expired
+      * it rewards the msg.sender with P % of the proposal's upstakes upon a successful call to this function.
+      * P = t/10 and t = number of blocks passed between the proposal's timeout and a successful call to this function.
+      * the function assume an avergae block time of 15 seconds.
```

Comment:
That seems to me just an unnecesaary complex representation of what you are doing.
Just  say "P = t/150, where t is the number of seconds passed since the the proposal's timeout", no need to make assumptions on block times (which you are not making, anyway, this code will run the same whatever the block times are)

Reply:
the rewards steps increments each on each 15 seconds(which is an average block time).
it is not continuous (step on each second). if we decided to do that we could ,but currently it is not.

Reply:
Yes, it increments on each 15 seconds, which is the average block time, so it will miscalculate the amount of blocks mined exactly half of the time.

To me it looks like adding complexity without any good reason at all. Time in the infra is measured everywhere in seconds and not in (approximations of) blocks, so why make an exception here?

Reply:
made it a linear function.  
see 382daa5